packages/formstr-app/src/containers/CreateFormNew/components/SectionManager/SectionEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Button, Modal, Form, Input, Typography } from 'antd';
+import { EditOutlined } from '@ant-design/icons';
+import useFormBuilderContext from '../../hooks/useFormBuilderContext';
+import { SectionData } from '../../providers/FormBuilder/typeDefs';
+
+const { TextArea } = Input;
+const { Text } = Typography;
+
+interface SectionEditorProps {
+  sectionId: string;
+}
+
+const SectionEditor: React.FC<SectionEditorProps> = ({ sectionId }) => {
+  const { sections, updateSection } = useFormBuilderContext();
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [form] = Form.useForm();
+  
+  const section = sections.find(s => s.id === sectionId);
+  
+  if (!section) return null;
+  
+  const showModal = () => {
+    form.setFieldsValue({
+      title: section.title,
+      description: section.description
+    });
+    setIsModalVisible(true);
+  };
+  
+  const handleSave = () => {
+    form.validateFields().then(values => {
+      updateSection(sectionId, values);
+      setIsModalVisible(false);
+    });
+  };
+  
+  return (
+    <>
+      <Button 
+        type="text" 
+        icon={<EditOutlined />} 
+        onClick={showModal}
+        style={{ float: 'right' }}
+      >
+        Edit Section
+      </Button>
+      
+      <Modal
```

Comment:
where is this modal used? dont see any modal popping up in the createForm screen?